app/hydrator/services/create/stores/config-store.js

```diff
@@ -176,7 +176,12 @@ class HydratorPlusPlusConfigStore {
       const sanitize =  window.CaskCommon.CDAPHelpers.santizeStringForHTMLID;
       if (node.outputSchemaProperty) {
         try {
-          let outputSchema = JSON.parse(node.outputSchema);
+          let outputSchema 
+          if (angular.isArray(node.outputSchema)) {
+            outputSchema = JSON.parse(node.outputSchema[0].schema);
```

Comment:
i don't know why this is happening. The joiner plugin returns an array of objects rather than a string